server/build.gradle

```diff
@@ -190,6 +190,7 @@ dependencies {
   compileOnly project(':app-server')
   compileOnly group: 'org.bouncycastle', name: 'bcprov-jdk15on', version: '1.47'
   compileOnly group: 'org.eclipse.jetty.websocket', name: 'websocket-server', version: versions.jetty
+  compile group: 'org.eclipse.jetty.websocket', name: 'javax-websocket-server-impl', version: versions.jetty
```

Comment:
This should probably be `compileOnly` to avoid pulling down all of jetty. And move these dependencies in `jetty9` for packaging in prod, and `development-server` for the dev mode.

If you look for `org.eclipse.jetty.websocket` in all gradle files, you'll see what needs to be done.